app/src/main/java/org/sopt/at/presentation/home/component/RecommendContent.kt

```diff
@@ -0,0 +1,116 @@
+package org.sopt.at.presentation.home.component
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.lazy.itemsIndexed
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.skydoves.landscapist.ShimmerParams
+import com.skydoves.landscapist.glide.GlideImage
+import org.sopt.at.R
+import org.sopt.at.model.Content
+import org.sopt.at.ui.theme.ButtonTint
+import org.sopt.at.ui.theme.TextFieldBg
+import org.sopt.at.ui.theme.White
+
+@Composable
+fun RecommendContent(
+    modifier: Modifier = Modifier,
+    title: String,
+    isSupportRanking: Boolean,
+    isShowMoreButton: Boolean,
+    imageUrls: List<String>
+) {
```

Comment:
텍스트를 보여주는 flag로 사용하기 위해 isShowMoreButton을 넣어주고 있잖아요. 
요렇게 하는 방법말고 컴포저블 자체를 넘겨줄 수도 있습니다.
```suggestion
@Composable
fun RecommendContent(
    modifier: Modifier = Modifier,
    title: String,
    isSupportRanking: Boolean,
    moreButtonText: @Composable () -> Unit
    imageUrls: List<String>
) {
```
함수로 받은 컴포저블을 아래 
"if (isShowMoreButton) { // 더보기 " 자리에 대신 넣어주면 됩니다.

Reply:
오호 새로 배워갑니다 감사합니다!